### DISCUSSION TITLE
iOS rendering order problem?

### DISCUSSION THREAD

**Discussion by danilokrulj** [google/model-viewer#3792] [Q&A]
Hi guys, I have a model of a table that has a glass top, I have planes inside the table which can be seen through the glass. The problem is that planes with the same material don't render the same in AR, the bottom plane is seen in AR but the side planes aren't rendered... I will attach photos to better explain the issue. 

The problem occurs on iOS devices. Did anyone encounter this problem and have they found any solution? Any help/tip is welcome.

![E22F389C-360E-4C86-A6D1-0BC573335C6B](https://user-images.githubusercontent.com/92090684/189104021-90e9455d-5e13-48bc-b9e3-27779a4c858f.png)
![1DA0844F-BF72-4345-83AD-5AAEDFDBE754](https://user-images.githubusercontent.com/92090684/189104024-f744bde7-5bd3-42fa-9a36-a7f039186316.png)

**Comment by elalish** [comment]
This looks like a problem with converting glTF to USDZ on the fly; we do our best, but the formats are not 100% compatible. You can also try using Apple's Reality Converter to create your USDZ and set that as your `ios-src`. FYI @grorg (Apple).

**Reply by danilokrulj** [reply]
Figured it was a conversion problem, I just couldn't understand why one plane gets rendered correctly and the others don't, considering they all use the same material with the same settings. Hope someone that had this issue and figured it out reads this :)
